### ISSUE TITLE
bold/color/weight for text in specific word in fig/ax.text

### ISSUE THREAD

**Issue by JosephBARBIERDARNAL** [holtzy/The-Python-Graph-Gallery#301]

**Comment by JosephBARBIERDARNAL** [comment]
## colors

It seems complicated:
- https://stackoverflow.com/questions/9169052/partial-coloring-of-text
- [Matplotlib doesn't natively support complex formatting of strings, such as partially colouring a label](https://www.futurile.net/2016/03/14/partial-colouring-text-in-matplotlib-with-latex/)

## weight and style

```
import matplotlib.pyplot as plt
plt.rcParams.update({
    "text.usetex": True,
    "text.latex.preamble": r"\usepackage{xcolor}",
})

fig, ax = plt.subplots()

# Add annotation with "world" in bold
ax.text(0.5, 0.5, r"hello $\mathbf{bold!}$", fontsize=12)
ax.text(0.1, 0.5, r"hello $\mathit{italic!}$", fontsize=12)

plt.show()
```

**Comment by holtzy** [comment]
Check highlight_text library!

**Comment by JosephBARBIERDARNAL** [comment]
fix with #321
